backend/migrations/20230703132445_image_rating.up.sql

```diff
@@ -0,0 +1,9 @@
+-- Add up migration script here
+CREATE DOMAIN vote as smallint CHECK (VALUE = -1 OR VALUE = 1);
```

Comment:
DEFAULT val = 0 for images without user vote?

Reply:
no vote == no tuple in relation